.github/workflows/mingw.yml

```diff
@@ -70,32 +71,30 @@ jobs:
         with:
           ruby-version: ${{ matrix.baseruby }}
 
-      - name: where check
+      - name: Misc system & package info
+        working-directory:
         run: |
           # show where
-          mv /c/Windows/System32/libcrypto-1_1-x64.dll /c/Windows/System32/libcrypto-1_1-x64.dll_
-          mv /c/Windows/System32/libssl-1_1-x64.dll    /c/Windows/System32/libssl-1_1-x64.dll_
           result=true
           for e in gcc.exe ragel.exe make.exe libcrypto-1_1-x64.dll libssl-1_1-x64.dll; do
             echo ::group::$'\033[93m'$e$'\033[m'
             where $e || result=false
             echo ::endgroup::
           done
-          $result
-        working-directory:
-
-      - name: version check
-        run: |
+          mv /c/Windows/System32/libcrypto-1_1-x64.dll /c/Windows/System32/libcrypto-1_1-x64.dll_
+          mv /c/Windows/System32/libssl-1_1-x64.dll    /c/Windows/System32/libssl-1_1-x64.dll_
           # show version
-          result=true
           for e in gcc ragel make "openssl version"; do
             case "$e" in *" "*) ;; *) e="$e --version";; esac
             echo ::group::$'\033[93m'$e$'\033[m'
             $e || result=false
             echo ::endgroup::
           done
+          # show packages
+          echo ::group::$'\033[93m'Packages$'\033[m'
+          pacman -Qs mingw-w64-ucrt-x86_64-* | grep local | sed "s/local\/mingw-w64-ucrt-x86_64-//"
```

Comment:
```suggestion
          pacman -Qs mingw-w64-ucrt-x86_64-* | sed "/local/!d;s,local/mingw-w64-ucrt-x86_64-,,"
```

Or maybe?
```suggestion
          pacman -Qs mingw-w64-ucrt-x86_64-* | sed -n "s,local/mingw-w64-ucrt-x86_64-,,p"
```

Reply:
Thanks.  Updated code using the 2nd and rebased